gulpfile.js

```diff
@@ -318,15 +216,14 @@ function fontello() {
       assetsOnly: false
     }))
     .pipe(print())
-    .pipe(gulp.dest('modules/core/client/fonts/fontello'))
-    .pipe(plugins.refresh());
+    .pipe(gulp.dest('modules/core/client/fonts/fontello'));
 }
 
 function mocha(done) {
   // Open mongoose connections
   const mongooseService = require('./config/lib/mongoose');
   const agenda = require('./config/lib/agenda');
-  const testSuites = changedTestFiles.length ? changedTestFiles : 'modules/*/tests/server/**/*.js';
+  const testSuites = changedTestFiles.length > 0 ? changedTestFiles : 'modules/*/tests/server/**/*.js';
```

Comment:
I switched for clarity, I can never quite remember what is truthy or not, and this seemed clearer to me. I changed it when I was confused what was going on (as it wasn't only re-running the changed files...). This change in itself though doesn't make any practical difference.

Reply:
Some folks (and linter presets like `xo`) prefer to check length against zero instead of truthy because in theory `object.length` could be whatever. Not a biggie either way. :-)